internal/controllers/apis/cluster/registry.go

```diff
@@ -0,0 +1,18 @@
+package cluster
+
+import (
+	"github.com/tilt-dev/tilt/internal/container"
+	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+)
+
+func RegistryHosting(registry *container.Registry) *v1alpha1.RegistryHosting {
```

Comment:
fwiw, we should eventually move towards getting rid of container.Registry and use the api struct everywhere